inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
I'd say that wanting to disable this is such an uncommon occurrence that it should not be optimized for. Optimizing for it has costs, as is pretty clear in this example.